iceoryx_hoofs/include/iceoryx_hoofs/error_handling/error_handling.hpp

```diff
@@ -17,289 +17,26 @@
-    error(PORT_POOL__APPLICATIONLIST_OVERFLOW) \
-    error(PORT_POOL__NODELIST_OVERFLOW) \
-    error(PORT_POOL__CONDITION_VARIABLE_LIST_OVERFLOW) \
-    error(PORT_POOL__EVENT_VARIABLE_LIST_OVERFLOW) \
-    error(PORT_MANAGER__PORT_POOL_UNAVAILABLE) \
-    error(PORT_MANAGER__INTROSPECTION_MEMORY_MANAGER_UNAVAILABLE) \
-    error(PORT_MANAGER__HANDLE_PUBLISHER_PORTS_INVALID_CAPRO_MESSAGE) \
-    error(PORT_MANAGER__HANDLE_SUBSCRIBER_PORTS_INVALID_CAPRO_MESSAGE) \
-    error(PORT_MANAGER__HANDLE_CLIENT_PORTS_INVALID_CAPRO_MESSAGE) \
-    error(PORT_MANAGER__HANDLE_SERVER_PORTS_INVALID_CAPRO_MESSAGE) \
-    error(PORT_MANAGER__NO_PUBLISHER_PORT_FOR_INTERNAL_SERVICE) \
-    error(ROUDI_COMPONENTS__SHARED_MEMORY_UNAVAILABLE) \
-    error(ROUDI_APP__FAILED_TO_CREATE_SEMAPHORE) \
-    error(ROUDI_APP__FAILED_TO_UNLOCK_SEMAPHORE_IN_SIG_HANDLER) \
-    error(ROUDI__DEFAULT_ROUDI_MEMORY_FAILED_TO_ADD_SEGMENT_MANAGER_MEMORY_BLOCK) \
-    error(ROUDI__DEFAULT_ROUDI_MEMORY_FAILED_TO_ADD_INTROSPECTION_MEMORY_BLOCK) \
-    error(ROUDI__PRECONDITIONS_FOR_PROCESS_MANAGER_NOT_FULFILLED) \
-    error(ICEORYX_ROUDI_MEMORY_MANAGER__COULD_NOT_ACQUIRE_FILE_LOCK) \
-    error(ICEORYX_ROUDI_MEMORY_MANAGER__ROUDI_STILL_RUNNING) \
-    error(ICEORYX_ROUDI_MEMORY_MANAGER__FAILED_TO_ADD_PORTPOOL_MEMORY_BLOCK) \
-    error(ICEORYX_ROUDI_MEMORY_MANAGER__FAILED_TO_ADD_MANAGEMENT_MEMORY_BLOCK) \
-    error(IPC_INTERFACE__UNABLE_TO_CREATE_APPLICATION_CHANNEL) \
-    error(IPC_INTERFACE__REG_ROUDI_NOT_AVAILABLE) \
-    error(IPC_INTERFACE__REG_UNABLE_TO_WRITE_TO_ROUDI_CHANNEL) \
-    error(IPC_INTERFACE__REG_ACK_INVALIG_NUMBER_OF_PARAMS) \
-    error(IPC_INTERFACE__REG_ACK_NO_RESPONSE) \
-    error(IPC_INTERFACE__CHECK_MQ_MAPS_TO_FILE) \
-    error(IPC_INTERFACE__APP_WITH_SAME_NAME_STILL_RUNNING) \
-    error(IPC_INTERFACE__COULD_NOT_ACQUIRE_FILE_LOCK) \
-    error(BINDING_C__UNDEFINED_STATE_IN_IOX_QUEUE_FULL_POLICY) \
-    error(BINDING_C__UNDEFINED_STATE_IN_IOX_CONSUMER_TOO_SLOW_POLICY) \
-    error(BINDING_C__PUBLISHER_OPTIONS_NOT_INITIALIZED) \
-    error(BINDING_C__SUBSCRIBER_OPTIONS_NOT_INITIALIZED) \
-    error(BINDING_C__C2CPP_ENUM_TRANSLATION_INVALID_SUBSCRIBER_EVENT_VALUE) \
-    error(BINDING_C__C2CPP_ENUM_TRANSLATION_INVALID_SUBSCRIBER_STATE_VALUE) \
-    error(BINDING_C__C2CPP_ENUM_TRANSLATION_INVALID_CLIENT_EVENT_VALUE) \
-    error(BINDING_C__C2CPP_ENUM_TRANSLATION_INVALID_CLIENT_STATE_VALUE) \
-    error(BINDING_C__C2CPP_ENUM_TRANSLATION_INVALID_SERVER_EVENT_VALUE) \
-    error(BINDING_C__C2CPP_ENUM_TRANSLATION_INVALID_SERVER_STATE_VALUE) \
-    error(BINDING_C__C2CPP_ENUM_TRANSLATION_INVALID_SERVICE_DISCOVERY_EVENT_VALUE) \
-    error(BINDING_C__C2CPP_ENUM_TRANSLATION_INVALID_MESSAGING_PATTERN_VALUE) \
+#define HOOFS_ERRORS(error) \
     error(EXPECTS_ENSURES_FAILED)
 
     // EXPECTS_ENSURES_FAILED is used as a temporary solution to make Expects/Ensures testable
 
 // clang-format on
 
-// DO NOT TOUCH START, you can doodle around with the lines above!!!
```

Comment:
Please re-add this comment or something similar. It must be ensured that the enum is not changed manually.